code/style.css

```diff
@@ -1,5 +1,371 @@
-/* If you can see a peach background, the css file is correctly linked. Feel free to change or remove this styling! */
+@import url('https://fonts.googleapis.com/css2?family=Bungee+Shade&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Pathway+Gothic+One&display=swap');
+
+/* menu */
+
+menu {
+    display: flex;
+    justify-content: center;
+    font-family: 'Pathway Gothic One', sans-serif;
+    padding: 0px 20px;
+
+  }
+  
+  menu a {
+    display: inline-block;
+    padding: 10px 15px;
+    font-size: 20px;
+    background: white;
+    color: black;
+    font-weight: bold;
+    text-decoration: none;
+    border-radius: 3px;
+    margin: 0 5px;
+    
+  }
+  
+  menu a:hover {
+    background:lightpink;
+    color: gray;
+  }
+
+/* general */
+
+header {
+    background-image: url(https://images.unsplash.com/photo-1559251606-c623743a6d76?ixlib=rb-1.2.1&q=85&fm=jpg&crop=entropy&cs=srgb&ixid=eyJhcHBfaWQiOjE0NTg5fQ);
+    height: 500px;
+    background-size: cover;
+    padding-top: 150px;
+    padding-bottom: 70px;
+    font-family: 'Pathway Gothic One', sans-serif;
+}
+
+h1 {
+    margin: 0px;
+    color: gray;
+    font-size: 90px;
+    text-align: center;
+    font-family: 'Bungee Shade', cursive;
+    padding-left: 10px;
+}
```

Comment:
Some of your `h1`'s are too wide for the page in mobile view. Consider a smaller font-size on smaller screens